docs/developer-docs/smart-contracts/signatures/threshold-schnorr.mdx

```diff
@@ -0,0 +1,37 @@
+---
```

Comment:
consistent file and ref names?

-> `threshold-schnorr.mdx`, `t-ecdsa.mdx`

Reply:
Renamed file for consistency